robot/impl/local_robot_test.go

```diff
@@ -4239,3 +4239,104 @@ func TestMaintenanceConfig(t *testing.T) {
 		test.That(t, sensorUnBlocked, test.ShouldNotBeNil)
 	})
 }
+
+func TestRemovingOfflineRemotes(t *testing.T) {
+	logger := logging.NewTestLogger(t)
+	ctx := context.Background()
+	cfg := config.Config{
+		Remotes: []config.Remote{
+			{
+				Name:     "remoteOffline",
+				Insecure: true,
+				Address:  "123.123.123.123",
+			},
+		},
+	}
+	r := setupLocalRobot(t, ctx, &cfg, logger)
+	time.Sleep(2 * time.Second)
```

Comment:
do all the sleeps need to exist? we want to use as few sleeps in tests as possible. this one shouldn't need to be here as far as I can tell

Reply:
Im still reworking the test, Its not quite testing exactly what I want it to test